Database/Corrections/QuestieEvent.lua

```diff
@@ -58,6 +73,13 @@ function QuestieEvent:Load()
     QuestieEvent.eventDates["Lunar Festival"] = QuestieEvent.lunarFestival[year]
     local activeEvents = {}
 
+    local eventCorrections = Questie.IsTBC and QuestieEvent.eventDateCorrections["TBC"] or QuestieEvent.eventDateCorrections["CLASSIC"]
+    for eventName,dates in pairs(eventCorrections) do
+        if dates then
+            QuestieEvent.eventDates[eventName] = dates
```

Comment:
You set `["Brewfest"] = false,` below so `dates = false`.

The next loop should break because of this, since it iterates over `QuestieEvent.eventDates` and wants to extract two values. Won't `eventData.startDate` will be `nil` and then errors on `eventData.startDate`?

Reply:
`if dates then` -> it never sets `QuestieEvent.eventDates[eventName] = false`

Reply:
Oh right, but how is that then removing Brewfest for Classic Era?

Reply:
Ah nevermind. Found the `and (eventCorrections[eventName] ~= false)` 🤦🏻

Reply:
This first block was just to change dates to correct ones.

Non-existing events are removed in next changed block:
`if _WithinDates(startDay, startMonth, endDay, endMonth) and (eventCorrections[eventName] ~= false) then`

There was some reason to not skip it fully at first changed block, but I can't remember anymore why. It would have created possible bug